mm2src/coins/utxo/utxo_standard.rs

```diff
@@ -47,7 +45,8 @@ pub async fn utxo_standard_coin_from_conf_and_request(
     req: &Json,
     priv_key: &[u8],
 ) -> Result<UtxoStandardCoin, String> {
-    let inner = try_s!(utxo_arc_from_conf_and_request(ctx, ticker, conf, req, priv_key, UTXO_STANDARD_DUST).await);
+    let inner = try_s!(utxo_common::utxo_arc_from_conf_and_request(ctx, ticker, conf, req, priv_key).await);
+    // TODO consider to move it into utxo_common.rs
```

Comment:
TODO

Reply:
I thought this snippet can be moved later, but I forgot about QTUM coin...

Reply:
I decided to not call `merge_utxo_loop` for Qrc20Coin, because it can be confusing if the `merge_utxo_loop` is called by QtumCoin